commonmark-ext-heading-anchor/src/main/java/ext/heading/anchor/HeaderIdExtension.java

```diff
@@ -0,0 +1,45 @@
+package ext.heading.anchor;
```

Comment:
Sorry, I wasn't clear. I meant only to change the part after the `org.commonmark`. The package should be `org.commonmark.ext.heading.anchor`.